theseus_gui/src/App.vue

```diff
@@ -252,6 +251,10 @@ list().then(
 }
 
 .settings {
+  svg {
+    color: var(--color-base) !important;
+  }
+
```

Comment:
This was needed for the settings icon in light-theme. Without it, it doesn't look quite right and seems to conflict with the router active style. I do feel it was a bit lazy, but wanted to get something quick in there since we'll be revisiting the sidebar sometime (when we do the expandable piece).